providers/register/provider_slurm.go

```diff
@@ -0,0 +1,21 @@
+// +build slurm_provider
```

Comment:
Should we add copyright here? 
Or copyright should be removed from `slurm` provider?